ext/native-decls/SetTrainStopAtStations.md

```diff
@@ -0,0 +1,18 @@
+---
+ns: CFX
+apiset: client
+game: gta5
+---
+## SET_TRAIN_STOP_AT_STATIONS
+
+
+```c
+void SET_TRAIN_STOP_AT_STATIONS(Vehicle train, BOOL state)
+
+```
+
+Toggles a train's ability to stop at stations
+
+## Parameters
+* **vehicle**: The train handle
+* **state**: True to make the train stop at stations. False to make train resume default behavior
```

Comment:
Nit: I assume theoretically this native can also be used with regular trains, not only the `CREATE_MISSION_TRAIN` ones. E.g. to make those trains not stop at the station.

Lets change the wording to remove the `resume default behavior` part. E.g. to something like `True to make the train stop at stations. False to make the train go without stops`